### PR TITLE
Adds dependencies to the README.md

### PR DESCRIPTION
**Pull request by officialgupta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,3 +143,24 @@ func main() {
...
+* github.com/go-gl/gl/v3.2-core/gl
+* github.com/go-gl/glfw/v3.2/glfw
+* github.com/golang/freetype
+* github.com/srwiley/oksvg
+* github.com/srwiley/rasterx
+* golang.org/x/image/font
+
+These can be included in your gopath by the following 'get' commands:
+
+* "go get github.com/go-gl/gl/v3.2-core/gl"
```
Comment (jadr2ddude) [inline README.md]: This is not necessary. A `go get github.com/fyne-io/fyne/...` would pull in all of these

File: README.md [hunk 2]
```diff
@@ -143,3 +143,24 @@ func main() {
...
+* "go get github.com/go-gl/glfw/v3.2/glfw"
+* "go get github.com/golang/freetype"
+* "go get github.com/srwiley/oksvg"
+* "go get github.com/srwiley/rasterx"
+* "go get golang.org/x/image/font"
+
+You will also require a GCC compiler if you do not have one, this can be included by installing TDM-GCC.
```
Comment (jadr2ddude) [inline README.md]: This assumes a Windows user, which is not necessarily the case. This could very well be a Darwin/Linux/*BSD/other user.

Also, TDM-GCC is not necessarily the best way to handle Windows CGO support (I had not even heard of it before now). If we even think that we need to explain this, we should link to the Go wiki: https://github.com/golang/go/wiki/WindowsBuild

File: README.md [hunk 3]
```diff
@@ -143,3 +143,24 @@ func main() {
 
 The main examples have been moved - you can find them in their [own repository](https://github.com/fyne-io/examples/).
 
+## Non-standard library dependencies
```
Comment (jadr2ddude) [inline README.md]: We could just rephrase this as "dependencies". This would be more succinct.

**Comment by jadr2ddude** [comment]
We may also want to think about whether we need to document these at all. If we are going to document these, then we will need to remember to update this when adding or removing dependencies.

File: README.md (hunk 1 above)
Comment (officialgupta) [reply README.md]: In my experience it has never pulled in these dependencies

Comment (andydotxyz) [reply README.md]: Was that back when you needed to pass -tags gl? Since we moved this to the default driver it should indeed be downloading the dependencies at the same time.

I will try on a fresh installation and see what happens.

File: README.md (hunk 2 above)
Comment (andydotxyz) [reply README.md]: I wonder if this could be phrased more along the lines of "Fyne uses CGo, which requires a C compiler to be installed. If you don't already have one installed you can ...".
After all it's not like we require a C compiler directly so it would be good if this sounds like less of a special case.

Comment (jadr2ddude) [reply README.md]: Also, we may be able to remove CGo dependency on windows in the near future :smile:

https://github.com/go-gl/glow/pull/102

**Comment by andydotxyz** [comment]
Hmm, @officialgupta is right - the dependencies are not pulled automatically. I would prefer that we can fix that problem rather than list the dependencies in documentation - if possible.

The CGo portion of this is still important - perhaps we can follow two different threads?
I have created #53 to track the install problem.

I'm sorry this PR has not simply been accepted - what do you think about just focussing on the C portion, with the expectation that #53 will be resolved?

File: README.md (hunk 1 above)
Comment (andydotxyz) [reply README.md]: You're right it does not work.
However, instead of listing the dependencies how about we do this instead? (as it will always work, even when the deps change)

```bash
go get -u github.com/fyne-io/fyne
cd $GOPATH/src/github.com/fyne-io/fyne
go get -u ./...
```

File: README.md [hunk 4]
```diff
@@ -143,3 +143,24 @@ func main() {
...
+These can be included in your gopath by the following 'get' commands:
+
+* "go get github.com/go-gl/gl/v3.2-core/gl"
+* "go get github.com/go-gl/glfw/v3.2/glfw"
+* "go get github.com/golang/freetype"
+* "go get github.com/srwiley/oksvg"
+* "go get github.com/srwiley/rasterx"
+* "go get golang.org/x/image/font"
+
+You will also require a GCC compiler if you do not have one, if on Windows this can be included by installing TDM-GCC.
```
Comment (andydotxyz) [inline README.md]: I agree this GCC mention is probably needed, but can we structure it a little more to be about CGo is required and link to some document about setting this up. If I was new to developing with Windows I'm not sure that "instal TDM-GCC" would be enough detail for me to get up and running?

**Comment by andydotxyz** [comment]
With #53 resolved the dependency issue should have gone away, so it should work with no workaround now.

File: README.md (hunk 4 above)
Comment (officialgupta) [reply README.md]: I did some digging and the official Golang documentation states that for Windows C support to use minGW; however many people complain this solution doesn't work and to use TDM-GCC.  Hence I continued to incude it.

What exactly would be required for the CGo details? This shouldn't be a problem if users do a full Go install, usually issues witch CGo arise when a GCC compiler doesn't exist on the system.

**Comment by officialgupta** [comment]
> With #53 resolved the dependency issue should have gone away, so it should work with no workaround now.

These have been removed in c/311b544

**Comment by andydotxyz** [comment]
Thanks for your work on this :)
For CGo it is always there but may not work without the C compiler. I guess this covers it for Windows users. Personally I use mingw_w64 which seems to work OK.
